frontend/src/assets/icons/PhHandsClapping.tsx

```diff
@@ -0,0 +1,15 @@
+// hands-clapping icon taken from  https://phosphoricons.com/.
```

Comment:
comment: sad now we are mixing icon sets 😆

Reply:
xd